modules/keyboardshortcut/keyboardshortcut.js

```diff
@@ -90,6 +90,17 @@ const KeyboardShortcut = {
         enabled = value;
     },
 
+    /**
+     * Opens the {@KeyboardShortcutsDialog} dialog.
+     *
+     * @returns {void}
+     */
+    openDialog() {
+        APP.store.dispatch(toggleDialog(KeyboardShortcutsDialog, {
+            shortcutDescriptions: _shortcutsHelp
+        }));
+    },
```

Comment:
openDialog was created and made public so clicking the toolbar button to open the keyboard shortcuts dialog could be handled through middleware while minimizing changes to keyboardshortcuts.js.